.github/workflows/examples.yml

```diff
@@ -1,9 +1,13 @@
-name: examples
+name: Gno Examples CI Suite
```

Comment:
Keep this one short; "CI suite" is redundant.

I like "Test Examples"

Reply:
Agreed, renamed 🙏 

[d0fe708](https://github.com/gnolang/gno/pull/3316/commits/d0fe7083e0923616594c45ec79486ddbf74c325c)